### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by rileyjsumner** [TravisJenniges/AnnandaleBBSB#4]
Hey @TravisJenniges Project is up and running, I've been building it in React. When you get a chance, could you run the project and let me know if I'm on the right track! 
Thanks

**Comment by TravisJenniges** [comment]
Could you update the readme.md with instructions for running the project?

**Comment by rileyjsumner** [comment]
Yup, just sent it now.

**Comment by TravisJenniges** [comment]
Yeah, the homepage looks good.  I'm realizing just how much information is on that page and might come up with a breakout to simplify it.  But the left nav, space for a header up top and content pane is good.
